src/scene/main_scene.cpp

```diff
@@ -25,13 +34,13 @@ void MainScene::c_update() {
 
 void MainScene::s_setup() {
     //Setting up map
-    grid = new Grid("assets/maps/dungeon_test.txt");
+    grid = new Grid(chosen_map.c_str());
```

Comment:
Can make `Grid` constructor take in a `std::string` instead of a char buffer to avoid this `c_str` stuff.  Just a suggestion, though